Contextualisation/cloud-cfg.txt

```diff
@@ -15,18 +15,47 @@ write_files:
 
   - path: /home/ubuntu/airfoil/tasks.py
     content: |
+      import subprocess
+      import pandas as pd
+      from glob import glob
       from celery import Celery
-      app = Celery('tasks', broker='amqp://')
+      
+      app = Celery('tasks', broker='amqp://js:js@192.168.2.162/jsvhost', backend='redis://:js@192.168.2.162/0')
```

Comment:
This setting is to be updated. There should be safer solutions than explicitly containing authentication like this.